app/views/projects/index.html.erb

```diff
@@ -1,4 +1,17 @@
-<h1>Projects</h1>
-<% @projects.each do |project| %>
-  <h2><%= project.name %></h2>
-<% end %>
+<div class="container">
+  <br>
+  <h1>Projects</h1>
+  <br>
```

Comment:
Use CSS rather than breaks in general. Fine for now, though.

Reply:
I will. Thanks